src/js/components/LogRow.js

```diff
@@ -62,9 +59,8 @@ const LogRow = (props: Props) => {
     <div
       className={classNames("log-row", {highlight, even: index % 2 === 0})}
       style={Styler.row(dimens)}
-      onClick={(e) => {
-        handleClick(e)
-      }}
+      onClick={onClick}
+      onDoubleClick={onDoubleClick}
```

Comment:
Due to decoupling the row selection from the logDetail stack, we can fire the click handler twice on double click without issue. This makes selecting the rows feel much snappier now since we don't have to wait for the 250ms timeout.